### PR TITLE
[MW] Wotc infographic update

### PR DESCRIPTION
**Pull request by abelito75**
Added more info and the frame work for more which I'll do in the future. Pushing this up now so if Anomoly or another person wants to take charge and add useful stuff to wotc infographic they can have the most up to date stuff

### CODE REVIEW

**Comment by anom0ly** [comment]
Hey -

Can you post a screenshot of the infograph update? Also can you fix the merge conflicts on this one?

Thanks

**Comment by abelito75** [comment]
This pr has 2 parts (bad idea looking at it in retro spec) a tooltip aspect and a bug fix

Old version of the tooltip:
oldwotctooltip

New version of the tooltip:
wotctooltip

Before the fix: 
bug

After the fix:
unbugged

File: src/parser/monk/mistweaver/modules/spells/essences/WayOfTheCrane.js [hunk 1]
```diff
@@ -87,21 +87,33 @@ class WayOfTheCrane extends Analyzer {
...
       <AzeritePowerStatistic
         size="flexible"
         tooltip={(
-          <ul>
-          {arrayOfDamageSpells.map(spell => (
-            <li>{spell} did {this.customMap.get(spell).healing} healing, {this.customMap.get(spell).overheal} overhealing in {this.customMap.get(spell).casts/3} casts.</li>
-          ))}
-        </ul>
+          <div>
+            You were in WOTC mode for {this._wotcTime} which was {formatPercentage((this._wotcTime/fightLength))}% of the fight.  You can see all all the raw data from them sexy kicks and punches below.
```
Comment (anom0ly) [inline src/parser/monk/mistweaver/modules/spells/essences/WayOfTheCrane.js]: For this, can we update the wording to something like:
`<SpellLink id={SPELLS.WAY_OF_THE_CRANE.id} /> was active for {this._wotcTime}, which was {formatPercentage((this._wotcTime/fightLength))}% of the fight. The raw data, per spell is below.`

File: src/parser/monk/mistweaver/modules/spells/essences/WayOfTheCrane.js [hunk 2]
```diff
@@ -87,21 +87,33 @@ class WayOfTheCrane extends Analyzer {
...
-          <ul>
-          {arrayOfDamageSpells.map(spell => (
-            <li>{spell} did {this.customMap.get(spell).healing} healing, {this.customMap.get(spell).overheal} overhealing in {this.customMap.get(spell).casts/3} casts.</li>
-          ))}
-        </ul>
+          <div>
+            You were in WOTC mode for {this._wotcTime} which was {formatPercentage((this._wotcTime/fightLength))}% of the fight.  You can see all all the raw data from them sexy kicks and punches below.   
+            <ul>
+            {arrayOfDamageSpells.map(spell => (
+             <li>{spell} did {this.customMap.get(spell).healing} healing, {this.customMap.get(spell).overheal} overhealing in {this.customMap.get(spell).casts/3} casts.
```
Comment (anom0ly) [inline src/parser/monk/mistweaver/modules/spells/essences/WayOfTheCrane.js]: Can we format each of the healing and overhealing numbers here? The output looks a bit raw with just the full number listed.

Using something like `formatNumber()` around the healing and overhealing on this.

Comment (abelito75) [reply src/parser/monk/mistweaver/modules/spells/essences/WayOfTheCrane.js]: 100%

File: src/parser/monk/mistweaver/modules/spells/essences/WayOfTheCrane.js (hunk 1 above)
Comment (abelito75) [reply src/parser/monk/mistweaver/modules/spells/essences/WayOfTheCrane.js]: Yeah... We can remove the sexy

**Comment by abelito75** [comment]
After requested changes:
textchange

**Comment by abelito75** [comment]
Newest commit was adding the word seconds after the time unit. So it will read 
... was active for 105 seconds...

**Pull request merged**